bindings/setup.py

```diff
@@ -28,6 +28,7 @@
     "src/utils/src/num.c",
     "src/modules/src/controller_mellinger.c",
     "src/modules/src/power_distribution_quadrotor.c",
+    # "src/modules/src/power_distribution_flapper.c",
```

Comment:
The Python bindings might require some extra work such that they function correctly with the Flapper...
If the second power distribution is included, the build test fails...